networking/src/Mockable/Concurrent.hs

```diff
@@ -98,6 +120,24 @@ delay time = liftMockable $ Delay time
 sleepForever :: ( Mockable Delay m ) => m ()
 sleepForever = liftMockable SleepForever
 
+----------------------------------------------------------------------------
+-- MyThreadId mock and helper functions
+----------------------------------------------------------------------------
+
+data MyThreadId m t where
+    MyThreadId :: MyThreadId m (ThreadId m)
```

Comment:
It returns current thread id, right? Document it maybe?

Reply:
This whole module is almost undocumented. I'd probably do it if it was pointed out in my first PR, but the issue is done already, so I won't (since it's not related to the issue anyway).